### ISSUE TITLE
PDF Report should contain SasView Version Number (Trac #1179)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#1215]
I'm putting this in for 4.2, but it equally applies to 5.0.

A colleague here in ISIS was struggling to reload a .svs project created over 2 years ago in a previous position. At issue was which version of !SasView the project had been created in.

By trial-and-good-fortune we discovered that the project had been created in 4.0.1 and then discovered it involved a plugin model that was naturally missing. Recreating the plugin then allowed the project to load ok, but VERY slowly.

@krzywon will be delighted to hear that with the plugin 4.2.0Beta loads the project like greased lightning!

However, a moral from this story is that given model names, algorithms, and save states have, and may again in the future, change, it would be disciplined and professional of us to include the !SasView version number in the PDF Report.

Migrated from http://trac.sasview.org/ticket/1179
```json
{
    "status": "closed",
    "changetime": "2018-10-01T09:14:25",
    "_ts": "2018-10-01 09:14:25.572417+00:00",
    "description": "I'm putting this in for 4.2, but it equally applies to 5.0.\n\nA colleague here in ISIS was struggling to reload a .svs project created over 2 years ago in a previous position. At issue was which version of !SasView the project had been created in.\n\nBy trial-and-good-fortune we discovered that the project had been created in 4.0.1 and then discovered it involved a plugin model that was naturally missing. Recreating the plugin then allowed the project to load ok, but VERY slowly.\n\n@krzywon will be delighted to hear that with the plugin 4.2.0Beta loads the project like greased lightning!\n\nHowever, a moral from this story is that given model names, algorithms, and save states have, and may again in the future, change, it would be disciplined and professional of us to include the !SasView version number in the PDF Report.\n",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "Reporting Enhancements",
    "time": "2018-09-20T13:54:09",
    "component": "SasView",
    "summary": "PDF Report should contain SasView Version Number",
    "priority": "blocker",
    "keywords": "",
    "milestone": "SasView 4.2.0",
    "owner": "",
    "type": "enhancement"
}
```

**Comment by butlerpd** [comment]
Trac update at `2018/09/27 14:44:01`: **butler** commented:

Do not believe this is really a blocker for 4.2 and don't see that anybody has the bandwidth to add in reasonable time.  Should discuss -- either it is a blocker or we need to move to 4.3 where it can remain at "critical" level. Leaving this as "Critical" for now even though we should have nothing but blockers left.

**Comment by butlerpd** [comment]
Trac update at `2018/09/29 00:45:03`: 
* **butler** commented:

OK .. not really something that should stop a long overdue version 4.2, but since it was easy enough and is now a PR making this a blocker till the PR is merged and the ticket closed


* **butler** changed priority from "critical" to "blocker"

**Comment by butlerpd** [comment]
Trac update at `2018/09/30 22:04:59`: 
* **butler** commented:

Closed with merging of !SasView PR 185.  In merging forgot to add the appropriate magical `closes #1215`


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "new" to "closed"

**Comment by smk78** [comment]
Trac update at `2018/10/01 09:14:25`: **smk78** commented:

Downloaded Win64 Build 432 and checked. Version number is there in the report. Thanks Jeff!
